firefox-ios/Client/Frontend/Browser/MainMenu/MainMenuTelemetry.swift

```diff
@@ -6,6 +6,18 @@ import Foundation
 import Glean
 
 struct MainMenuTelemetry {
-    func mainMenuViewed() { }
-    func mainMenuDismissed() { }
+    func optionTapped(with isHomepage: Bool, and option: String) {
+        let extra = GleanMetrics.AppMenu.MainMenuOptionSelectedExtra(isHomepage: isHomepage, option: option)
+        GleanMetrics.AppMenu.mainMenuOptionSelected.record(extra)
+    }
+
+    func closeButtonTapped(isHomepage: Bool) {
+        let extra = GleanMetrics.AppMenu.CloseButtonExtra(isHomepage: isHomepage)
+        GleanMetrics.AppMenu.closeButton.record(extra)
+    }
+
+    func menuDismissed(isHomepage: Bool) {
+        let extra = GleanMetrics.AppMenu.MenuDismissedExtra(isHomepage: isHomepage)
+        GleanMetrics.AppMenu.menuDismissed.record(extra)
+    }
```

Comment:
We should have test for this stuff. I believe `MicrosurveyTelemetryTests` would be a good model to follow.